addons/modules/functions/fnc_moduleNukeLocal.sqf

```diff
@@ -0,0 +1,382 @@
+    "",
+    "Billboard",
+    1,
+    20,
+    [0, 0, 0],
+    [0, 0, 60],
+    0,
+    1.7,
+    1,
+    0,
+    [60, 80, 100],
+    [
+        [1, 1, 1, -10],
+        [1, 1, 1, -7],
+        [1, 1, 1, -4],
+        [1, 1, 1, -0.5],
+        [1, 1, 1, 0]
+    ],
+    [0.05],
+    1,
+    1,
+    "",
+    "",
+    _source
+];
+
+_top1 setParticleRandom [0, [75, 75, 15], [17, 17, 10], 0, 0, [0, 0, 0, 0], 0, 0, 360];
+_top1 setDropInterval 0.002;
+
+private _top2 = "#particlesource" createVehicleLocal [0, 0, 0];
+_top2 setPosASL _position;
+
+_top2 setParticleParams [
+    ["A3\Data_F\ParticleEffects\Universal\universal.p3d", 16, 3, 112, 0],
+    "",
+    "Billboard",
+    1,
+    20,
+    [0, 0, 0],
+    [0, 0, 60],
+    0,
+    1.7,
+    1,
+    0,
+    [60, 80, 100],
+    [
+        [1, 1, 1, 0.5],
+        [1, 1, 1, 0]
+    ],
+    [0.07],
+    1,
+    1,
+    "",
+    "",
+    _source
+];
+
+_top2 setParticleRandom [0, [75, 75, 15], [17, 17, 10], 0, 0, [0, 0, 0, 0], 0, 0, 360];
+_top2 setDropInterval 0.002;
+
+private _smoke = "#particlesource" createVehicleLocal [0, 0, 0];
+_smoke setPosASL _position;
+
+_smoke setParticleParams [
+    ["A3\Data_F\ParticleEffects\Universal\universal.p3d", 16, 7, 48, 1],
+    "",
+    "Billboard",
+    1,
+    25,
+    [0, 0, 0],
+    [0, 0, 60],
+    0,
+    1.7,
+    1,
+    0,
+    [40, 15, 120],
+    [
+        [1, 1, 1, 0.4],
+        [1, 1, 1, 0.7],
+        [1, 1, 1, 0.7],
+        [1, 1, 1, 0.7],
+        [1, 1, 1, 0.7],
+        [1, 1, 1, 0.7],
+        [1, 1, 1, 0.7],
+        [1, 1, 1, 0]
+    ],
+    [0.5, 0.1],
+    1,
+    1,
+    "",
+    "",
+    _source
+];
+
+_smoke setParticleRandom [0, [10, 10, 15], [15, 15, 7], 0, 0, [0, 0, 0, 0], 0, 0, 360];
+_smoke setDropInterval 0.002;
+
+private _wave = "#particlesource" createVehicleLocal [0, 0, 0];
+_wave setPosASL _position;
+
+_wave setParticleParams [
+    ["A3\Data_F\ParticleEffects\Universal\universal.p3d", 16, 7, 48],
+    "",
+    "Billboard",
+    1,
+    20,
+    [0, 0, 0],
+    [0, 0, 0],
+    0,
+    1.5,
+    1,
+    0,
+    [50, 100],
+    [
+        [0.1, 0.1, 0.1, 0.5],
+        [0.5, 0.5, 0.5, 0.5],
+        [1, 1, 1, 0.3],
+        [1, 1, 1, 0]
+    ],
+    [1, 0.5],
+    0.1,
+    1,
+    "",
+    "",
+    _source
+];
+
+_wave setParticleRandom [2, [20, 20, 20], [5, 5, 0], 0, 0, [0, 0, 0, 0.1], 0, 0];
+_wave setParticleCircle [50, [-80, -80, 2.5]];
+_wave setDropInterval 0.0002;
+
+private _light = "#lightpoint" createVehicleLocal [0, 0, 0];
+_light setPosASL (_position vectorAdd [0, 0, 500]);
+
+_light setLightAmbient [1, 0.8, 0.67];
+_light setLightColor [1, 0.8, 0.67];
+_light setLightBrightness 100;
+
+[{
+    15 fadeMusic _this;
+}, musicVolume, 2] call CBA_fnc_waitAndExecute;
+
+0 fadeMusic 1;
+
+[3] call EFUNC(common,earthquake);
+
+[{
+    enableCamShake true;
+    addCamShake [5, 10, 25];
+}, [], 2] call CBA_fnc_waitAndExecute;
+
+if (_colorCorrections) then {
```

Comment:
Full yellow screen effect suddenly goes away, would look better if it gradually fades.